public/manifest.json

```diff
@@ -1,23 +1,7 @@
 {
   "short_name": "React App",
   "name": "Create React App Sample",
```

Comment:
- [ ] **[OPTIONAL]** I would recommend you remove this `manifest.json` file generated by CRA, since you would not be using it so that your codebase looks more neat.
- [ ] **[OPTIONAL]** I would recommend you remove the `public/robots.txt` file generated by CRA.